src/main/java/com/spider/solomon/order/OrderItemDao.java

```diff
@@ -0,0 +1,31 @@
+package com.spider.solomon.order;
+
+import java.util.List;
+
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.stereotype.Repository;
+
+import jakarta.annotation.Resource;
+
+@Repository
+public class OrderItemDao {
```

Comment:
If we make the change to eliminate the N+1 problem mentioned in an other comment, we can go ahead and delete `getOrderItemsByOrderId` as it would be unused.

We could also consider deleting the entire `OrderItemDao` and reintroducing it later if needed, or we could keep it to preserve the `RowMapper` if you wanted to compose this mapper with the one in `CustomerOrderDao`